html/canvas/element/manual/imagebitmap/createImageBitmap-colorSpaceConversion.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<title>createImageBitmap + drawImage test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/html/canvas/resources/canvas-tests.js"></script>
+<script src="/common/media.js"></script>
+<script src="common.sub.js"></script>
+<link rel="stylesheet" href="/html/canvas/resources/canvas-tests.css">
+<body>
+<script>
+function testCanvasDisplayingPattern(canvas, width, height, options)
+{
+    var tolerance = 10; // high tolerance for differing colour management results
+    const check = (x, y, r, g, b, a) =>
+        _assertPixelApprox(canvas, x,y, r,g,b,a, `${x},${y}`, `${r},${g},${b},${a}`, tolerance);
+
+    if (options.colorSpaceConversion == 'none') {
+        check(1 * width / 4, 1 * height / 4, 124,0,27,255);
+        check(3 * width / 4, 1 * height / 4, 0,124,46,255);
+        check(1 * width / 4, 3 * height / 4, 60,0,123,255);
+        check(3 * width / 4, 3 * height / 4, 0,0,0,255);
+    } else {
```

Comment:
I don't know a ton about canvas, so forgive the naive question, but [the spec states](https://html.spec.whatwg.org/#:~:text=If%20val%20is%20%22default%22%2C%20the%20color%20space%20conversion%20behavior%20is%20implementation%2Dspecific):

> If val is "default", the color space conversion behavior is implementation-specific, and should be chosen according to the default color space that the implementation uses for drawing images onto the canvas.

Should we be asserting implementation-specific behavior in this clause?

Reply:
You are right, checks for default option removed.